### ISSUE TITLE
crash peerbadsignature

### ISSUE THREAD

**Issue by ThomasWaldmann** [nsupdate-info/nsupdate.info#266]
```
Internal Server Error: /domain/701/
Traceback (most recent call last):
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 132, in get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 71, in view
    return self.dispatch(request, *args, **kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 34, in _wrapper
    return bound_func(*args, **kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/contrib/auth/decorators.py", line 22, in _wrapped_view
    return view_func(request, *args, **kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/utils/decorators.py", line 30, in bound_func
    return func.__get__(self, type(self))(*args2, **kwargs2)
  File "/srv/nsupdate.info/repo/nsupdate/main/views.py", line 465, in dispatch
    return super(DomainView, self).dispatch(*args, **kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/views/generic/base.py", line 89, in dispatch
    return handler(request, *args, **kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 272, in post
    return super(BaseUpdateView, self).post(request, *args, **kwargs)
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/views/generic/edit.py", line 214, in post
    if form.is_valid():
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/forms/forms.py", line 184, in is_valid
    return self.is_bound and not self.errors
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/forms/forms.py", line 176, in errors
    self.full_clean()
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/forms/forms.py", line 393, in full_clean
    self._clean_form()
  File "/srv/nsupdate.info/env/local/lib/python2.7/site-packages/django/forms/forms.py", line 417, in _clean_form
    cleaned_data = self.clean()
  File "/srv/nsupdate.info/repo/nsupdate/main/forms.py", line 71, in clean
    check_domain(self.instance.name)
  File "/srv/nsupdate.info/repo/nsupdate/main/dnstools.py", line 114, in check_domain
    add(fqdn, "8.8.8.8")
  File "/srv/nsupdate.info/repo/nsupdate/main/dnstools.py", line 154, in add
    update_ns(fqdn, rdtype, ipaddr, action=action, ttl=ttl)
  File "/srv/nsupdate.info/repo/nsupdate/main/dnstools.py", line 372, in update_ns
    raise DnsUpdateError("PeerBadSignature")
DnsUpdateError: PeerBadSignature

Request repr(): 
<WSGIRequest
path:/domain/701/,
GET:<QueryDict: {}>,
POST:<QueryDict: {u'comment': [u''], u'nameserver_update_secret': [u'N.............................................='], u'nameserver2_ip': [u'92.51.xxx.xxx'], u'nameserver_ip': [u'92.51.xxx.xxx'], u'csrfmiddlewaretoken': [u'v...................l'], u'available': [u'on'], u'nameserver_update_algorithm': [u'HMAC_SHA512']}>,
COOKIES:{'csrftoken': 'v..................l',
 'sessionid': '7..................p'},
META:{'CONTENT_LENGTH': '289',
 'CONTENT_TYPE': 'application/x-www-form-urlencoded',
 u'CSRF_COOKIE': u'v....................l',
 'DOCUMENT_ROOT': '/srv/nsupdate.info/htdocs',
 'HTTP_ACCEPT': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
 'HTTP_ACCEPT_ENCODING': 'gzip, deflate, br',
 'HTTP_ACCEPT_LANGUAGE': 'de-DE,de;q=0.8,en-US;q=0.6,en;q=0.4',
 'HTTP_CACHE_CONTROL': 'max-age=0',
 'HTTP_CONNECTION': 'close',
 'HTTP_CONTENT_LENGTH': '289',
 'HTTP_CONTENT_TYPE': 'application/x-www-form-urlencoded',
 'HTTP_COOKIE': 'sessionid=7......................p; csrftoken=v.....................l',
 'HTTP_HOST': 'www.nsupdate.info',
 'HTTP_ORIGIN': 'https://www.nsupdate.info',
 'HTTP_REFERER': 'https://www.nsupdate.info/domain/701/',
 'HTTP_UPGRADE_INSECURE_REQUESTS': '1',
 'HTTP_USER_AGENT': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/51.0.2704.84 Safari/537.36',
 'HTTP_X_FORWARDED_FOR': '217.86.xx.xx',
 'HTTP_X_FORWARDED_PROTO': 'https',
 'HTTP_X_REAL_IP': '217.86.xx.xx',
 'PATH_INFO': u'/domain/701/',
 'QUERY_STRING': '',
 'REMOTE_ADDR': '217.86.xx.xx',
 'REMOTE_PORT': '',
 'REQUEST_METHOD': 'POST',
 'REQUEST_URI': '/domain/701/',
 u'SCRIPT_NAME': u'',
 'SERVER_NAME': 'www.nsupdate.info',
 'SERVER_PORT': '80',
 'SERVER_PROTOCOL': 'HTTP/1.0',
 'UWSGI_APPID': 'nsupdate',
 'UWSGI_SCHEME': 'https',
 'uwsgi.core': 5,
 'uwsgi.node': 'web.thinkmo.de',
 'uwsgi.version': '2.0.7-debian',
 'wsgi.errors': <open file 'wsgi_errors', mode 'w' at 0x7f3f221be270>,
 'wsgi.file_wrapper': <built-in function uwsgi_sendfile>,
 'wsgi.input': <uwsgi._Input object at 0x7f3f24301b88>,
 'wsgi.multiprocess': False,
 'wsgi.multithread': True,
 'wsgi.run_once': False,
 'wsgi.url_scheme': 'https',
 'wsgi.version': (1, 0)}>
```

**Comment by elnappo** [comment]
should be already fixed with https://github.com/nsupdate-info/nsupdate.info/commit/f987fae7e0935ba79f97c900de607467c0240b83. Is this already live?

**Comment by ThomasWaldmann** [comment]
oh, oops. thanks for pointing that out. i really should upgrade the live system...

**Comment by elnappo** [comment]
This should also fix #257 and #258

**Comment by ThomasWaldmann** [comment]
likely fixed, and live meanwhile.
